.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   tests:
-    uses: centralnicgroup-opensource/rtldev-middleware-shareable-workflows/.github/workflows/auto-merge-dependabot-pr.yml@main
+    uses: centralnicgroup-opensource/rtldev-middleware-shareable-workflows/.github/workflows/auto-merge-dependabot-pr.yml@RSRMID-1665-workflows-fail-as-of-upload-download-artifacts-v-4-upgrade
```

Comment:
I think, this can't be merged because of the branch name being part of uses reference.
Or is it for testing purposes?

Reply:
Yes, correct, that's just for testing purpose.